forem/forem
# Adding a modal for the CSV export

## What type of PR is this? (check all applicable)


## Description
In [an older PR](https://github.com/forem/forem/pull/17217), I built the CSV functionality on the backend. Now, I've added the user experience as per the [designs](https://www.figma.com/file/Ejc7VELuN00n1SZrkUTBcQ/Member-List-View?node-id=708%3A5065). 

i.e. I've added a modal with some information and a download button.
 
## Related Tickets & Documents
- Related Issue #https://github.com/forem/forem/issues/16934
- Closes #https://github.com/forem/forem/issues/16934

## QA Instructions, Screenshots, Recordings
1. Go to https://www.loom.com/share/16f2ead870614452ba85c1c775369b38
2. Click on the export icon 
3. Make sure that a modal opens
4. Click on the download button and ensure that the CSV downloads

https://www.loom.com/share/16f2ead870614452ba85c1c775369b38

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [y] Yes
      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/oNrQaV8zjY5RD7Ok67/giphy-downsized-large.gif)
